src/caselawclient/Client.py

```diff
@@ -753,6 +771,19 @@ def get_properties_for_search_results(self, judgment_uris: list[str]) -> str:
         response = self._send_to_eval(vars, "get_properties_for_search_results.xqy")
         return decode_multipart(response)
 
+    def search_and_parse_results(
+        self, search_parameters: SearchParameters
+    ) -> SearchResults:
+        response = self.advanced_search(search_parameters)
+        response_string = decode_multipart(response)
+        return SearchResults(response_string)
+
+    def search_judgments_and_parse_results(
```

Comment:
Had started updating `perform_advanced_search` in public ui to filter by `collections="judgment"`, but I think now we abstract this here we should keep a general `search_and_parse_results` (equivalent to perform_advanced_search`) but also create a judgment specific one as we currently only care about judgment - thus this method is created :)